generated/nidmm/session.py

```diff
@@ -186,6 +196,7 @@ def __init__(self, resource_name, id_query=0, reset_device=False, options_string
         self.library = library.get_library()
         self.vi = 0  # This must be set before calling _init_with_options.
         self.vi = self._init_with_options(resource_name, id_query, reset_device, options_string)
+        self.acquisition = Acquisition(self)
```

Comment:
intersting. I don't think it's appropriate for acquisition to be a property, it should be a method since it is semantically an action.